frontend/src/app/components/containers/complaints/outcomes/supplemental-notes/supplemental-notes-item.tsx

```diff
@@ -66,7 +68,14 @@ export const SupplementalNotesItem: FC<props> = ({ notes, action, enableEditMode
         </div>
         <div className="comp-details-right-column">
           <CompTextIconButton
-            id="notes-edit-button"
```

Comment:
Removing the notes-edit-button id from the Edit button is causing some of the Cypress tests to be skipped as the edit button can't be found.